CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -705,7 +705,7 @@ public function postProcess() {
               }
               else {
                 CRM_Core_DAO::setFieldValue('CRM_Price_DAO_PriceField', $priceFieldID, 'is_required', 0);
-                CRM_Core_DAO::setFieldValue('CRM_Price_DAO_PriceFieldValue', $priceFieldValueID, 'label', 'Other Amount');
+                CRM_Core_DAO::setFieldValue('CRM_Price_DAO_PriceFieldValue', $priceFieldValueID, 'label', ts('Other Amount'));
```

Comment:
@kurund I'm not sure I understand. Labels are always translated in the DB. For multi-lingual, we have label_fr_CA, label_en_US, etc.